browser/js/friend/friend.html

```diff
@@ -4,6 +4,7 @@
   <md-button class="md-raised md-primary" ng-click="follow(friend.id)" ng-show="userFriendsIds.indexOf(friend.id) === -1">Follow {{friend.name}}</md-button>
   <md-button class="md-raised" ng-show ="userFriendsIds.indexOf(friend.id) !== -1" ng-click="unfollow(friend.id)">Unfollow {{friend.name}}</md-button>
   <h4>Learning:</h4>
+  <md-button class="md-raised md-primary" ng-diabled=true ng-show ="userFriendsIds.indexOf(friend.id) !== -1">You Follow {{friend.name}}</md-button>
```

Comment:
That line is a mistake I just took it out, when I fixed the merge conflict it was still in there.  Your toggle button code is directly above that, the functionality is already there.